index.html

```diff
@@ -1648,6 +1607,76 @@
           </table>
         </section>
       </section>
+      <section>
+        <h3>
+          Interface <a><code>Presentation</code></a>
+        </h3>
+        <pre class="idl">
+          partial interface Navigator {
+            [SameObject] readonly attribute Presentation presentation;
+          };
+
+</pre>
+        <p>
+          The <dfn for="Navigator"><code>presentation</code></dfn> attribute is
+          used to retrieve an instance of the <a>Presentation</a> interface.
+        </p>
+        <pre class="idl">
+          interface Presentation {
+            // This API used by controlling browsing context.
+            attribute PresentationRequest? defaultRequest;
+
+            // This API is available on presenting browsing context.
+            [SameObject]
+            readonly attribute PresentationReceiver? receiver;
+          };
+
+</pre>
+        <p>
+          The <dfn for="Presentation"><code>defaultRequest</code></dfn> MUST
+          return the <a>default presentation request</a> if any, <code>null</code>
+          otherwise.
```

Comment:
On the other hand what is the effect of setting `defaultRequest` on a UA that implements the receiver but not the controller?  I assume that it's a no-op, i.e. the associated algorithms would not execute.